yb-voyager/src/srcdb/mysql.go

```diff
@@ -174,6 +177,58 @@ func (ms *MySQL) FilterEmptyTables(tableList []*sqlname.SourceName) ([]*sqlname.
 	return nonEmptyTableList, emptyTableList
 }
 
+func (ms *MySQL) GetTableColumns(tableName *sqlname.SourceName) ([]string, []string) {
+	var columns, datatypes []string
+	query := fmt.Sprintf("SELECT COLUMN_NAME, DATA_TYPE from INFORMATION_SCHEMA.COLUMNS where table_schema = '%s' and table_name='%s'", ms.source.DBName, tableName.ObjectName.Unquoted)
+	rows, err := ms.db.Query(query)
+	if err != nil {
+		utils.ErrExit("failed to query %q for finding table columns: %v", query, err)
+	}
+	for rows.Next() {
+		var column, dataType string
+		err := rows.Scan(&column, &dataType)
+		if err != nil {
+			utils.ErrExit("failed to scan column name from output of query %q: %v", query, err)
+		}
+		columns = append(columns, column)
+		datatypes = append(datatypes, dataType)
+	}
+	return columns, datatypes
+}
+
+func (ms *MySQL) PartiallySupportedTablesColumnList(tableList []*sqlname.SourceName, useDebezium bool) (map[string][]string, []string) {
```

Comment:
Why the word "Partially" in the name? Can't we have something like `GetColumnsWithSupportedTypes(tableList, useDebezium)`